Flows/HomeFlow/Sources/HomeFlow/PostItemLoadingView/PostItemLoadingView.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  [NAME].swift
+//  StackOv ([NAME] module)
+//
+//  Created by Влад Климов
+//  Copyright © 2021 Erik Basargin. All rights reserved.
+//
+
+import SwiftUI
+import Palette
+import Components
+
+struct PostItemLoadingView: View {
+
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let minWidth: CGFloat = 267
+        static let minHeight: CGFloat = 223
+        static let lightItemColor = Color(red: 0.873, green: 0.873, blue: 0.892)
+        static let darkItemColor = Color(red: 0.231, green: 0.235, blue: 0.255)
```

Comment:
Need to check this colours with the design (+ their names) and to declare them in the Palette module 🙃
To use them in the View layer, please add their short versions `fileprivate extension Color` which called as their purpose.